lib/rester/service/resource/params.rb

```diff
@@ -189,6 +212,14 @@ def _validate_method(key, obj, opt, value)
       def _error!(message)
         Errors.throw_error!(Errors::ValidationError, message)
       end
+
+      def _merge_params(params)
+        @_validators      = @_validators.merge(params.validators)
+        @_defaults        = @_defaults.merge(params.defaults)
+        @_required_fields = @_required_fields | params.required_params
+        @_all_fields      = @_all_fields | params.all_fields
+        self
```

Comment:
Why not return `nil`?

So you can do something like?
```ruby
p = Params.new
p.use(...).use(...)
```

Trying to think of a use case for that. If we can think of one, then let's add a test to ensure that `use` always returns the Params object.  Otherwise, let's make it just `nil`.